pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -122,9 +122,12 @@ def work(self):
                                     logger.log('Catch Rate with normal Pokeball has increased to {}%'.format(success_percentage))
                                 else:
                                     if response_dict['status_code'] is 1:
-                                        logger.log('Fail to use berry. Seem like you are softbanned.', 'red')
+                                        logger.log('Failed to use berry. Seems like you are softbanned.', 'red')
+                                        logger.log('Heading to nearest Pokestop to attempt unban.', 'red')
+                                        self.bot.softbanned = True
+                                        break
```

Comment:
Why isn't this a return? If we are softbanned, what else is there to do in the catch worker?